### ISSUE TITLE
U+7D6B in the Chinese versions is with a incorrect glyph

### ISSUE THREAD

**Issue by eyeplum** [iaolo/Desktop-Backgrounds#1]
The Chinese version of the desktop backgrounds has a character represented incorrectly.

<img width="362" alt="screen shot 2018-06-27 at 4 02 20 pm" src="https://user-images.githubusercontent.com/988437/41952403-c32ac894-7a23-11e8-897d-b75a4c7f1f89.png">

Here the original text is: `十黍为一絫，十絫为一铢`, but in the picture the character `U+7D6B 絫` is represented by a incorrect glyph (possibly missing in the font?).

**Comment by iaolo** [comment]
You're right. And, yes, it's probable the font. Will pick a proper Chinese font instead. Thanks for pointing it out. Always nervous when working with languages I don't understand.
